### ISSUE TITLE
Reactive Feature

### ISSUE THREAD

**Issue by ghost** [HabitRPG/habitica#184]
This is just one reactive feature i would like to see in habitrpg
-   Reactive Habits
  -   The app should react to you. If you do a certain todo regularly then it should be automatically moved into dailies. In a likewise fashion should your habit be going red the app should automatically put that at the top of the list and add a little message to your todos "do xxx habit today".

**Comment by geebru** [comment]
I feel like you're missing the purpose of those two sections of Habit.
Habits are things you do (or want to do) everyday, but aren't large tasks. For example my habits are drinking a glass of water (for XP!) and biting my nails (ouch, HP drain).
Dailies are (for me) eating breakfast and going to bed before 12:30. They ARE technically habits, but I will only do those once per day, and if I don't, I want to be docked for it.
To-do are one off things that have to happen, again in my example, getting a maintenance order form in for my broken dryer.
It's up to you to classify your things and no amount of logic could take what YOU have put in and prioritize it without more people finding it doing it when it shouldn't happen.

TL;DR - Habits are things you do multiple times a day, dailies are once a day, and to-do's are one offs. I don't think we need to complicate things with reactive logic. Of course, with better examples I might be swayed to join your side :P

**Comment by wc8** [comment]
I'm doing "reactive habits" manually. 
Many tasks would improve my quality of life and productivity if I did them every day or every week, but it would quickly overwhelm me to try to succeed at everything at once. 
So I started with an absolute minimum (medicines, 5 meals, X minutes exercise) for health and 2 minimum goals for work. Everything else (cleaning, filing, scanning, etc.) I put in the habits section. 
As I see progress in the daily section, I add in a couple from the habits section (or from my brain that I remember I should add in). 
I [also think it is best] a user maintained switch. 
The most I think HabitRPG might be able to do is give you a challenge notice: "You're getting good at exercise, would you like to make it a daily goal?" or "Your progress in Task X is abysmal, want to challenge yourself with a daily habit?" The user needs to be able to say NO though, because there are habits that are simply inappropriate to do make once per day tasks.

**Comment by ghost** [comment]
@wc8 This is very true. It was just an idea.

**Comment by horusofoz** [comment]
@Neohuman As it appears "Reactivity" to the degree you're looking for is currently outside of the scope of HabitRPG's plans, are you cool with closing this ticket?

**Comment by ghost** [comment]
For the time being yes but ultimately this will be implemented

**Comment by lefnire** [comment]
only if it's what many people want

**Comment by ghost** [comment]
Good point

**Comment by horusofoz** [comment]
@Neohuman Not sure we need the ticket reopened. If more users are interested in the feature they'll still be able to comment in the thread, prompting yourself or a mod to re-open. Coolies?

**Comment by ghost** [comment]
Coolies
